packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1304,6 +1304,77 @@ export const UnknownFilesystemError = {
 	hint: 'It can be caused by many things, eg. missing permissions or a file not existing we attempt to read. Check the error cause for more details.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Cannot extract the font type from the given URL.
+ */
+export const CannotExtractFontType = {
+	name: 'CannotExtractFontType',
+	title: 'Cannot extract the font type from the given URL.',
+	hint: 'Open an issue at https://github.com/withastro/astro/issues',
+} satisfies ErrorData;
+
+/**
+ * @docs
+ * @description
+ * Cannot fetch the given font file
+ * @message
+ * An error occured while fetching font file from the given URL.
+ */
+export const CannotFetchFontFile = {
+	name: 'CannotFetchFontFile',
+	title: 'Cannot fetch the given font file.',
+	message: (url: string) => `An error occured while fetching font file from ${url}`,
+	hint: 'Check the error cause and open an issue if it cannot be solved on your side.',
```

Comment:
```suggestion
	hint: 'Check that you have the correct URL to the font file. Please open an issue if Astro is unable to fetch a valid font file.',
```
Is it something like this?

Reply:
The URL is coming from a font provider so it's hidden to the user. IMO if the fetch fails it will mostly be because of connectivity issues (either on the user side or the font api eg. google)